uisdk/src/main/java/com/karhoo/uisdk/screen/rides/detail/RideDetailView.kt

```diff
@@ -35,7 +34,30 @@ import com.karhoo.uisdk.util.DateUtil
 import com.karhoo.uisdk.util.IntentUtils
 import com.karhoo.uisdk.util.extension.toLocalisedString
 import com.squareup.picasso.Picasso
-import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.*
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.baseFareIcon
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.bookingTermsText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.carText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.cardLogoImage
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.cardNumberText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.commentsLayout
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.commentsText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.dateTimeText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.dropOffLabel
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.flightDetailsLayout
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.flightNumberText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.karhooId
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.logoImage
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.meetingPointText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.pickupLabel
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.pickupTypeText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.priceText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.priceTypeText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.rebookRideButton
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.reportIssueButton
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.rideDetailCancellationText
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.starRatingWidget
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.stateIcon
+import kotlinx.android.synthetic.main.uisdk_view_ride_detail.view.stateText
```

Comment:
I think that the wild-card imports rule from detekt is a bit exaggerated 😁

Reply:
> Blindly importing all the classes in a package clutters the class namespace and could lead to conflicts between classes in different packages with the same name. On the other hand, specifically listing the necessary classes avoids that problem and makes clear which versions were wanted.
https://rules.sonarsource.com/java/RSPEC-2208